fuzz/fuzz_targets/parse_expr.rs

```diff
@@ -0,0 +1,7 @@
+#![no_main]
```

Comment:
Why do you need that?

Reply:
This is generated code when you cargo fuzz add.   I don't know.